### PR TITLE
[CHORE] force resolution of transitive dependencies

### PR DESCRIPTION
**Pull request by BeeGodwin**
Fixing vulns with `npm audit fix` introduced some hard-to-resolve breaking changes in `chrome-launcher`. So, use a preinstall script to force upgrading the transitive dependencies that were causing clients of a11y-tests-web to fail CI audit checks.

Also, migrate from `harp-minify` to `minify`-we were using the harp fork from back when minify itself wasn't on npm, but it hasn't kept up.

### CODE REVIEW

**Comment by BeeGodwin** [comment]
Worth noting that the preinstall script used here would not be needed with yarn, as yarn supports resolutions natively.

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "node": ">=10.0.0"
   },
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
```
Comment (andymsuk) [inline package.json:13]: Have you investigated using [shrinkwrap](https://docs.npmjs.com/cli/shrinkwrap)?

Comment (BeeGodwin) [reply package.json:13]: Good shout- done in 5bbf4aa. I think we still need the npx script though as we want the transitive dependencies upgraded without bumping the direct dependencies.

Comment (BeeGodwin) [reply package.json:13]: hmm, though I now see Travis chokes on the npx step.

**Comment by BeeGodwin** [comment]
Didn't merge as the root cause here is we need a new Lighthouse. (6.1.1).

**Pull request closed without merging**